app/models/manager_refresh/inventory_collection.rb

```diff
@@ -107,8 +107,10 @@ class InventoryCollection
              :find_by,
              :lazy_find,
              :lazy_find_by,
+             :named_ref,
              :primary_index,
              :reindex_secondary_indexes!,
+             :skeletal_primary_index,
```

Comment:
What's the difference between the skeletal_primary_index and the primary_index? If we're using skeletal precreate can we just use the same index?

Reply:
it's separate index now, since the processing is different. E.g. skeletal records are processed with ON CONFLICT DO NOTHING sql